src/caches/low_storage_rk_caches.jl

```diff
@@ -1371,3 +1371,71 @@ end
 function alg_cache(alg::ParsaniKetchesonDeconinck3S205,u,rate_prototype,uEltypeNoUnits,uBottomEltypeNoUnits,tTypeNoUnits,uprev,uprev2,f,t,dt,reltol,p,calck,::Type{Val{false}})
   ParsaniKetchesonDeconinck3S205ConstantCache(real(uBottomEltypeNoUnits),real(tTypeNoUnits))
 end
+
+
+
+# 2R+ low storage methods introduced by van der Hauwen
+@cache struct LowStorageRK2RPCache{uType,rateType,uNoUnitsType,TabType} <: OrdinaryDiffEqMutableCache
+  u::uType
+  k::rateType
+  Xᵢ::rateType
+  fsalfirst::rateType
+  tmp::uType
+  atmp::uNoUnitsType
+  tab::TabType
+end
+
+struct LowStorageRK2RPConstantCache{N,T,T2} <: OrdinaryDiffEqConstantCache
+  A₁::SVector{N,T}
+  Bₗ::T
+  B̂ₗ::T
+  Bᵢ::SVector{N,T}
+  B̂ᵢ::SVector{N,T}
+  Cᵢ::SVector{N,T2}
+end
+
+
+function CKLLDDRK43_2ConstantCache(::Type{T},::Type{T2}) where {T,T2}
+  A1 = convert(T, 0.32416573882874605)   #11847461282814//36547543011857
```

Comment:
I am not completely sure how many of the rational coefficients given in that article are exact solutions of the order conditions. Nevertheless, you should implement the rational forms, I think. They should at least provide some additional accuracy compared to the `Float64` forms.

Reply:
They are definitely helpful if anyone goes to higher order, since otherwise the float64 coefficients will just have accuracy dropoff

Reply:
while sometimes not practical, that can be helpful in internal benchmarks and testing.

Reply:
The problem with using rational coefficients is that the values of `c` is not specifically mentioned, so I had to calculate `c` values by adding up row elements in the butcher table. But when adding rational coefficients, the numbers go beyond `Int64` range. So I converted all of them to `Float64`. and mentioned their rational values next to them.

Reply:
You could use bigger integer values, e.g. `Int128(11847461282814)//36547543011857`. Does that help?